packages/babel-preset-env/test/fixtures/corejs2/entry-ie-11/output.mjs

```diff
@@ -1,13 +1,18 @@
+import "core-js/modules/es6.array.concat.js";
```

Comment:
In corejs2, there is not a `modules/es6.array.concat.js`. Is this a bug?
https://github.com/zloirock/core-js/tree/v2/modules